library.properties

```diff
@@ -0,0 +1,10 @@
+name=Azure SDK for C
+version=1.0.0-beta.1
+author=Microsoft Corporation
+maintainer=Microsoft <iotcert@microsoft.com>
+sentence=Azure SDK for C library (1.3.0-beta.1) for Arduino.
+paragraph=Arduino port of the Azure SDK for C. It allows you to use your Arduino device with Azure services like Azure IoT Hub and Azure Device Provisioning Service. See README.md for more details. Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+category=Communication
```

Comment:
I will assume this is the appropriate category based on past precedence. The categories here don't give much description:
https://arduino.github.io/arduino-cli/0.20/library-specification/#libraryproperties-file-format

Reply:
Yes, that's the recommended category for our SDK.